core-model-test/framework/src/main/java/org/jboss/as/core/model/test/TestModelControllerService.java

```diff
@@ -170,35 +172,27 @@ public void start(StartContext context) throws StartException {
             initializer.setRootResourceDefinitionDelegate();
         }
         super.start(context);
+        //todo super hack
+        if (type == TestModelType.STANDALONE){
+            ServerEnvironmentService.addService(((ServerInitializer)initializer).environment, context.getChildTarget());
+        }
+
     }
 
     @Override
     protected void initCoreModel(ManagementModel managementModel, Resource modelControllerResource) {
-        super.initCoreModel(managementModel, modelControllerResource);
+        //super.initCoreModel(managementModel, modelControllerResource);
         if (modelInitializer != null) {
             modelInitializer.populateModel(managementModel);
         }
-    }
-
-    @Override
-    protected void initCoreModel(Resource rootResource, ManagementResourceRegistration rootRegistration, Resource modelControllerResource) {
-        //See server HttpManagementAddHandler
-        System.setProperty("jboss.as.test.disable.runtime", "1");
-        if (type == TestModelType.STANDALONE) {
-            initializer.initCoreModel(rootResource, rootRegistration, modelControllerResource);
-
-        } else if (type == TestModelType.HOST){
-            initializer.initCoreModel(rootResource, rootRegistration, modelControllerResource);
-
-        } else if (type == TestModelType.DOMAIN){
-            initializer.initCoreModel(rootResource, rootRegistration, modelControllerResource);
-        }
+        System.setProperty("wildfly.test.boot.disable.rollback", "1");
```

Comment:
This got out of sync when I reverted the WFCORE-307 fix. The property is "jboss.as.test.disable.runtime".